internal/slackcontext/slackcontext.go

```diff
@@ -12,6 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package slackcontext defines getters and setters for request-scoped values
```

Comment:
```suggestion
// Package slackcontext defines getters and setters of invocation-scoped values
```

Nit: IMO "request" is an unfamiliar term in this context 😉

Reply:
Commit 521ae6f updates this to "execution-scoped" to match the other references to a command execution :)